.github/workflows/dockerfile-llb-frontend.yaml

```diff
@@ -0,0 +1,36 @@
+name: tools/dockerfile-llb-frontend
+
+on:
+  push:
+    branches: [stable]
+    paths:
+      - 'tools/dockerfile-llb-frontend/**'
+      - '.github/workflows/dockerfile-llb-frontend.yaml'
```

Comment:
This should be modified such that it builds for each PR and always builds for each merge into `stable`:

```yaml
on:
  push:
    branches: [stable]
  pull_request:
    types: [opened, synchronize, reopened]
    branches: [staging]
    paths:
      - 'tools/dockerfile-llb-frontend/**'
      - '.github/workflows/dockerfile-llb-frontend.yaml'
```

Reply:
Thanks